components/common/contracts-shortcuts.ts

```diff
@@ -30,6 +32,8 @@ export async function accountBalances(account: EthAccount, contracts: Contracts)
     erc1155BalanceOf(account.address, contracts.debtCouponToken),
     erc1155BalanceOf(account.address, (contracts.bondingToken as unknown) as ERC1155Ubiquity),
     contracts.usdc.balanceOf(account.address),
+    erc1155BalanceOf(account.address, (contracts.uBOND as unknown) as ERC1155Ubiquity), // uBOND
+    erc1155BalanceOf(account.address, contracts.uDEBT), // uDEBT
```

Comment:
This returns the total balance as a BigNumber sum. But for the UI we'll need each token differentiated; check how the `erc1155BalanceOf` is created, and you'll see we first get a list of IDs for an address, and then we call the balanceOf of each token ID. We need the ID and the balance both, so we can then use it to sell these tokens individually.
Then we'll also need to expiration block, but we'll add a pin to that as we need to check how to get it from the DebtCoupon contract.